### DISCUSSION TITLE
Cannot use o1

### DISCUSSION THREAD

**Discussion by KamiPasi** [open-webui/open-webui#5934] [General]
# Bug Report

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment

- **Open WebUI Version:** ghcr.io/open-webui/open-webui:main:v0.3.30

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Description

**Bug Summary:**

Unsupported value: 'messages[0].role' does not support 'system' with this model. (request id: 202410061522394929628uvlDaUA2) (request id: 20241006152238886535171kVpje3wZ)

## Logs and Screenshots

![image](https://github.com/user-attachments/assets/8fbd0c6d-bc48-4065-8d77-b6e0e87c71b4)

![image](https://github.com/user-attachments/assets/98f0a5ac-a5bf-423c-99ee-e9e88c73bfd2)

**Comment by YuriiKosiy** [comment]
![IMG_0335](https://github.com/user-attachments/assets/2a02b217-2642-4241-abf2-a3aab1779c05)
I have the same error on version 0.3.30 as well.

**Reply by TianzeWang** [reply]
Same issue.
